src/components/addToList.js

```diff
@@ -0,0 +1,15 @@
+import renderData from './renderData';
+import saveToStorage from './saveToStorage';
+
+const addToList = () => {
+  const newTask = document.forms[0].elements['task-input'].value;
+
+  document.forms[0].elements['task-input'].value = '';
+
+  saveToStorage(newTask);
+  renderData();
+
+  document.getElementsByClassName('bottom-area')[0].style.visibility = 'visible';
+};
+
```

Comment:
- [ ] When a task is added, the `index` attribute of the task should be equal to the length of the array. So if there are 3 tasks in the localStorage, their respective indexes should be `1`, `2`, `3`, instead of `0`, `1`, `2`. Kindly fix that.

  ![2we](https://user-images.githubusercontent.com/36542986/185089882-8bbd16f3-d26f-4758-81c1-54cc8b1ba839.jpg)